### PR TITLE
Index routing functional 2

### PR DESCRIPTION
**Pull request by chaitanyaphalak**

### CODE REVIEW

File: test/k8s_logging_tests/test_config.py [hunk 1]
```diff
@@ -47,3 +48,105 @@ def test_cluster_name(setup, test_input, expected):
...
+
+    logging.getLogger().info("Sleeping for 60 seconds")
+    time.sleep(60)
+
+    # Check if we have those generated logs from kubernetes in Splunk
+    v1.delete_namespaced_pod(name=image_name, namespace=namespace, body=pod)
+
+    search_query = "index={0}".format(test_input)
+    events = check_events_from_splunk(index=index,
+                                      start_time="-24h@h",
```
Comment (luckyj5) [inline test/k8s_logging_tests/test_config.py]: please update the time to -1h@h

File: test/k8s_logging_tests/test_config.py [hunk 2]
```diff
@@ -47,3 +48,105 @@ def test_cluster_name(setup, test_input, expected):
...
+    #                               url=setup["splunkd_url"],
+    #                               user=setup["splunk_user"],
+    #                               password=setup["splunk_password"]):
+    #     pytest.fail("Test failed as splunk index %s wasn't created" % index)
+
+    # Handle special cases of default namespaces kube-system and kube-public
+    if test_input == "kube-system" or test_input == "kube-public":
+        search_query = "index={0}".format(test_input)
+        events = check_events_from_splunk(index=index,
+                                          start_time="-24h@h",
```
Comment (luckyj5) [inline test/k8s_logging_tests/test_config.py]: update time to -1h@h

File: test/k8s_logging_tests/test_config.py (hunk 1 above)
Comment (chaitanyaphalak) [reply test/k8s_logging_tests/test_config.py]: cool

File: test/k8s_logging_tests/test_config.py (hunk 2 above)
Comment (chaitanyaphalak) [reply test/k8s_logging_tests/test_config.py]: cool

**Pull request merged**